src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -564,15 +564,21 @@ export class BrowsingContextImpl {
     };
   }
 
+  /**
+   * Issues `Page.addScriptToEvaluateOnNewDocument` CDP command with the given
+   * script source in evaluated form and world name / sandbox.
+   *
+   * @return The CDP preload script ID.
+   */
   async addPreloadScript(
```

Comment:
Looks like this method should be moved to BrowingcontextProcessor, as it handles on the CDP's target level.